### PR TITLE
Add testcase for manager_test.go

### PR DESCRIPTION
**Pull request by amitkumarj441**
**What this PR does / why we need it**: New testcase on deviceplugin for manager_test.go

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by amitkumarj441** [comment]
@RenaudWasTaken Can you have a look at this PR?

**Comment by RenaudWasTaken** [comment]
Hello @amitkumarj441 !

I'm not sure testing the return value of `m.CheckpointFile()` against a hardcoded string is a good test.
In this specific case as long as `m.CheckpointFile()` returns a constant I don't think it's worth testing it.

File: pkg/kubelet/cm/devicemanager/manager_test.go [hunk 1]
```diff
@@ -147,6 +147,13 @@ func TestDevicePluginReRegistration(t *testing.T) {
 	close(callbackChan)
 }
 
+func TestCheckpointFile(t *testing.T) {
```
Comment (yue9944882) [inline pkg/kubelet/cm/devicemanager/manager_test.go]: Do we really need this?

**Pull request closed without merging**